src/testdrive/src/action.rs

```diff
@@ -98,12 +99,13 @@ pub struct State {
     aws_region: rusoto_core::Region,
     aws_account: String,
     aws_credentials: AwsCredentials,
-    kinesis_client: KinesisClient,
+    pub kinesis_client: KinesisClient,
     kinesis_stream_names: Vec<String>,
     s3_client: S3Client,
     s3_buckets_created: BTreeSet<String>,
     sqs_client: SqsClient,
     sqs_queues_created: BTreeSet<String>,
+    pub default_timeout: f64,
```

Comment:
(Specific suggestion below, but state is meant to be an opaque struct within this module, so would prefer not to expose its fields!)